docs/master/eloquent/nested-mutations.md

```diff
@@ -512,6 +512,8 @@ input UpdatePostsHasMany {
   update: [UpdatePostInput!]
   upsert: [UpsertPostInput!]
   delete: [ID!]
+  connectMany: [ID!]
+  disconnectMany: [ID!]
```

Comment:
```suggestion
  connect: [ID!]
  disconnect: [ID!]
```

The fact that there are many is implied by nature of this being within the `HasMany` input and taking a list of IDs.

Reply:
I am sorry but i couldn't understand what you meant, would you please elaborate more.

Reply:
I propose to rename the fields as per my suggestion, mainly because it is consistent with other nested mutations.

Explicitly adding the `Many` suffix is redundant. For once, the field is already contained in an `input UpdatePostsHasMany` - that signifies that its function is to mutate a `HasMany` relation. Also, the value of the fields are a list `[ID!]`, that also signifies that the client can pass any number.

Reply:
I used `connectMany` and `disconnectMany` cuz in `ManyToMnay` relation it was calling static method `NestedOneToMany::createUpdateUpsert($args, $relation)` to `create`,  `update`, and `upsert`.

So I separated the logic into 2 methods so they won't interfere with each other and won't impact the `ManyToMnay` relation.

`NestedOneToMany.php`
```php
    /**
     * @param  \Illuminate\Database\Eloquent\Model  $parent
     * @param ArgumentSet $args
     */
    public function __invoke($parent, $args): void
    {
        /** @var \Illuminate\Database\Eloquent\Relations\HasMany|\Illuminate\Database\Eloquent\Relations\MorphMany $relation */
        $relation = $parent->{$this->relationName}();

        static::createUpdateUpsert($args, $relation);
        static::connectDisconnect($args, $relation);

        if ($args->has('delete')) {
            $relation->getRelated()::destroy(
                $args->arguments['delete']->toPlain()
            );
        }
    }
```

Reply:
Nicely done, looks pretty clean.